Example/Tests/TestHelper.swift

```diff
@@ -30,3 +30,31 @@ class TestHelper: NSObject {
         TestHelper.shared.mockTracker.addMock(name, returnValue: returnValue)
     }
 }
+
+extension Dictionary where Key: Comparable {
+    /// Converts the dictionary to a string with keys in stable order.
+    /// - Returns: A string representation of the dictionary with sorted keys.
+    func toStringWithStableOrder() -> String {
+        let sortedKeyValuePairs = self.sorted { $0.key < $1.key }
+        let keyValuePairsString = sortedKeyValuePairs.map { "\($0.key): \($0.value)" }.joined(separator: ", ")
+        return "[\(keyValuePairsString)]"
+    }
+}
+
+extension Dictionary {
+    /// Maps the keys and values of the dictionary to their string representations.
+    /// - Returns: A new dictionary with string keys and values.
+    func mapKeyAndValuesToString() -> [String: String] {
+        var args = [String: String]()
+        for (key, value) in self {
+            args[String(describing: key)] = String(describing: value)
+        }
+        return args
+    }
+
+    /// Converts the dictionary to a string with keys in stable order.
+    /// - Returns: A string representation of the dictionary with sorted keys.
+    func toStringWithStableOrder() -> String {
+        mapKeyAndValuesToString().toStringWithStableOrder()
+    }
```

Comment:
This should not be needed again

Reply:
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/70/commits/3bbd8df5a7dfc32245033706327f249e3a37084f

Reply:
I double checked. I meant it as a convenience, the `[AnyHashable: Any]` has no comparable key, so we would need to call `mapKeyAndValuesToString().toStringWithStableOrder()` on all those maps.

So might be worth to have it...
Let me know what you think :)

Reply:
As discussed offline: Let's keep the convenience function.

https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/70/commits/2998b75fa2177af176a97b946818d8c7533511cd